consumer/dragonboard410c/build/kernel.md

```diff
@@ -5,11 +5,12 @@ permalink: /documentation/consumer/dragonboard410c/build/kernel.md.html
 
 # Build & Update Linux kernel for Debian/OpenEmbedded
   
-This page provides instructions for building Linux kernel on Dragonboard-410c
-for Debian/OpenEmbedded based systems.
+This page provides the instructions for building and deploying linux kernel
+on Dragonboard410c from x86 host machine.
 
 ##### Prerequisites: 
-- A Linux OS (debian, openembedded...) already running on the board (emmc).
+- A Linux OS (debian, openembedded...) already running on the Dragonboard410c (emmc).
+- x86 Host machine running Debian based operating system.
```

Comment:
this could confuse users , we don't 'require' debian on host, but what you meant is that instructions here are if user runs debian (we should probably say "or Ubuntu"). we might want to add instructions for Fedora as well

Reply:
@ndechesne Yeah. Since we specify `apt-get` for installing packages, this instruction set assumes Debian host. But anyway once we add Fedora or some other OS based instructions we can modify this text.

Regarding the common kernel doc, I thought that this is something which will happen during the common Dragonboard doc which @ric96 is working on!

Reply:
Agree with @ndechesne here.

Listing Debian as a pre-req at the top is unhelpful. The indication that the `apt-get` instructions apply to Debian (and derivatives such as Ubuntu) should appear only in the section where the packages are installed.

You can always add "For other distributions try installing the development packages needed to build a native kernel." to cover the distros you haven't been able to test on.